### PR TITLE
Docs added for login and otp molecules

### PR DESCRIPTION
**Pull request by prtkjakhar**

### CODE REVIEW

File: docs/molecules/loginMobileAadharPage.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Styling
+- CSS modules are used for styling the component.
+- Styles are defined in the index.module.css file.
+
+## Notes
+- This molecule does not handle actual authentication. It simulates the login process by displaying a success message upon clicking the login button.
+- The registration functionality is not implemented in this molecule.
+
+
```
Comment (geeky-abhishek) [inline docs/molecules/loginMobileAadharPage.md:94]: can you mention how to customise this ,how to update theme and how to add our own action

File: docs/molecules/otpPage.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Styling
+- CSS modules are used for styling the component.
+- Styles are defined in the index.module.css file.
+
+## Notes
+- This molecule does not handle actual OTP verification. It simulates the verification process by displaying a success message upon entering the correct OTP.
+- The OTP length is hardcoded to 4 digits.
+
+
```
Comment (geeky-abhishek) [inline docs/molecules/otpPage.md:120]: same for this

Comment (prtkjakhar) [reply docs/molecules/otpPage.md:120]: added

File: docs/molecules/loginMobileAadharPage.md (hunk 1 above)
Comment (prtkjakhar) [reply docs/molecules/loginMobileAadharPage.md:94]: added

**Pull request merged**